src/backend/patches/@mapeo+core+9.0.0-alpha.4+001+fix-quickbit-dynamic-require.patch

```diff
@@ -21,3 +21,68 @@ index d4cfdfd..41cc2db 100644
 +import quickbit from 'quickbit-universal'
  
  export { quickbit }
+diff --git a/node_modules/@mapeo/core/src/discovery/dns-sd.js b/node_modules/@mapeo/core/src/discovery/dns-sd.js
+index 677006d..707ae35 100644
+--- a/node_modules/@mapeo/core/src/discovery/dns-sd.js
++++ b/node_modules/@mapeo/core/src/discovery/dns-sd.js
+@@ -35,18 +35,30 @@ export class DnsSd extends TypedEmitter {
+   #error = null
+   /** @param {import('bonjour-service').Service} service */
+   #handleServiceUp = (service) => {
++    this.#log(service.addresses, 'addresses')
++    this.#log(service.port, 'port')
++    this.#log(service, "service")
++
+     if (service.name === this.#name) {
+       this.#log(`Ignoring service up from self`)
+       return
+     }
+-    const address = service.addresses?.filter(isIpv4)[0]
+-    /* c8 ignore start */
+-    if (!address) {
+-      this.#log(`service up (${service.name}) with no ipv4 addresses`)
++    // const address = service.addresses?.filter(isIpv4)[0]
++    const { name, port, referer } = service
++    if (!referer) {
++      this.#log(`service up (${service.name}) had no referer`)
+       return
+     }
++
++    const { address } = referer
```

Comment:
Evan and I discovered that `address` was showing up in `referer`, but not in `service`. @gmaclennan

Reply:
Hmm I have a feeling that the changes in this patch will break things in other network environments, but if it’s enough to get it working for initial testing then that’s ok for now.

Reply:
This doesn't change how the front end is being implemented, so if we need to revert it and fix it another way, it shouldn't make a difference